MCprep_addon/materials/generate.py

```diff
@@ -104,6 +104,8 @@ def get_mc_canonical_name(name: str) -> Tuple[str, Optional[Form]]:
 	if "_stem" in general_name:
 		# For melon/pumpkin stem stage map to only melon_stem/pumpkin_stem
 		general_name = general_name.rsplit("_", 1)[0]
+	if "short_grass" in general_name:
+		general_name = "grass"
```

Comment:
I think we actually don't need to do this, at least once we update the `mcprep_data_update.json`. Maybe we'd need to do the opposite.
![image](https://github.com/user-attachments/assets/2a4cad42-ea14-44e8-8987-ab1ce777c3ac)

Reply:
@zNightlord can you test with this `mcprep_data.json` file and see if this section would still work with OBJs from older worlds? Just make sure you don't commit the updated JSON to the branch.

[mcprep_data.json](https://github.com/user-attachments/files/19697457/mcprep_data.json)

Reply:
added the desaturated for "short_grass" in the base json and removed the above if.
Yeah it doesn't work since old worlds use "grass" and that caused by "grass": "colormap/grass" in the json. I have to manually restore deleted json and reload change it to work, I guess have to fix in [mcprep_data_refresh.py](https://github.com/Moo-Ack-Productions/MCprep/blob/6514aa5b9bf39c1241865c9fbefe8c0d150305c6/mcprep_data_refresh.py#L458) add it in the overrides.

Reply:
I think a better route then would just be to ignore the change, or check if `short_grass` is already in the JSON. IMO we should avoid removing texture names from the JSON file as much as possible